### PR TITLE
bump dependencies

### PR DESCRIPTION
**Pull request by jmcampanini**
bumping deps and using dropwizard 0.9.2, so we're checklist compliant

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -35,8 +35,7 @@ dependencies {
     testCompile "io.dropwizard:dropwizard-client:$dropwizardVersion"
     testCompile "io.dropwizard:dropwizard-testing:$dropwizardVersion"
     testCompile "junit:junit:$junitVersion"
-    testCompile "org.hamcrest:hamcrest-all:$hamcrestVersion"
```
Comment (qinfchen) [inline build.gradle:38]: remove this from gradle.properties?

File: gradle/publish.gradle [hunk 2]
```diff
@@ -35,6 +35,9 @@ bintray {
         licenses = ['Apache-2.0']
         vcsUrl = 'https://github.com/palantir/dropwizard-web-security.git'
         publications = ['bintray']
+
+        githubRepo = 'palantir/dropwizard-web-security'
+        githubReleaseNotesFile = 'README.md'
```
Comment (qinfchen) [inline gradle/publish.gradle:40]: why the README.md?

File: build.gradle (hunk 1 above)
Comment (jmcampanini) [inline build.gradle:38]: pretty sure i did, or maybe i missed one?

File: gradle/publish.gradle (hunk 2 above)
Comment (jmcampanini) [inline gradle/publish.gradle:40]: talked offline. can look here for reference: https://github.com/bintray/gradle-bintray-plugin#buildgradle

File: build.gradle (hunk 1 above)
Comment (jmcampanini) [inline build.gradle:38]: talked offline. looks good! :)

**Comment by jmcampanini** [comment]
verbal +1 from @qinfchen

**Pull request merged**